modules/conversion-center/assets/scss/widgets/contact-buttons.scss

```diff
@@ -264,4 +259,160 @@
+                border-width: 0 40px 40px 0;
+                left: unset;
+                right: -20px;
+                transform: rotate(-90deg);
+            }
+        }
+    }
+
+    &__message-bubble-name {
+        color: var(--e-contact-buttons-message-bubble-name);
+        font-size: 20px;
+        line-height: 25px;
+        font-weight: 600;
+        margin-block-end: 8px;
+    }
+
+    &__message-bubble-body {
+        color: var(--e-contact-buttons-message-bubble-body);
+        font-size: 20px;
+        line-height: 25px;
+        margin-block-end: 8px;
+    }
+
+    &__message-bubble-time {
+        color: var(--e-contact-buttons-message-bubble-time);
+        font-size: 20px;
+        line-height: 25px;
+        font-weight: 600;
+        text-align: right;
+
+        @if $direction == rtl {
+            text-align: left;
+        }
+    }
+
+    &__dots-container {
+        background-color: var(--e-contact-buttons-message-bubble-bubble-bg);
+        border-radius: 15px;
+        display: inline-flex;
+        padding: 10px 12px;
+    }
+
+    &__dot {
+        animation: contactButtonsTypingJump 1s infinite;
+        background-color: var(--e-contact-buttons-message-bubble-name);
+        border-radius: 50%;
+        display: inline-block;
+        height: 7px;
+        margin-left: auto;
+        margin-right: 3px;
+        position: relative;
+        width: 7px;
+    }
+
+    &__dot-1 {
+        animation-delay: 200ms;
+    }
+    
+    &__dot-2 {
+        animation-delay: 400ms;
+    }
+    
+    &__dot-3 {
+        animation-delay: 600ms;
+    }
+
+    &__content.is-visible {
+
+        .e-contact-buttons__message-bubble.has-typing-animation {
+
+            .e-contact-buttons__dots-container {
+                animation-delay: 0;
+                animation-duration: 2s;
+                animation-fill-mode: forwards;
+                animation-iteration-count: 1;
+                animation-name: contactButtonsDisappear;
+            }
+        
+            .e-contact-buttons__bubble-container {
+                animation-delay: 2s;
+                animation-duration: .1s;
+                animation-fill-mode: forwards;
+                animation-iteration-count: 1;
+                animation-name: contactButtonsAppear;
+
+                height: 0;
+                opacity: 0;
+                visibility: hidden;
+            }
+        }
+    }
+}
+
+@keyframes contactButtonsTypingJump {
+    0% {
+        bottom: 0px;
+    }
+    20% {
+        bottom: 5px;
+    }
+    40% {
+        bottom: 0px;
+    }
+}
+
+@keyframes contactButtonsAppear {
+    from {
+        height: 0;
+        opacity: 0;
+        visibility: hidden;
+    }
+    to {
+        height: auto;
+        opacity: 1;
+        visibility: visible;
+    }
+}
+
+@keyframes contactButtonsDisappear {
+    from {
+        display: inline;
+        visibility: visible;
+    }
+    to {
+        display: none;
+        visibility: hidden;
+    }
```

Comment:
Why do we need `display` and `visibility`?

Reply:
we definitely need display none, as we need to hide the dots after a couple of seconds. I'm removing visibility